qcommon/test/org/qcmg/common/util/SnpUtilsTest.java

```diff
@@ -108,24 +108,6 @@ public void getBaseCountsACs() {
 		
 	}
 	
-	@Test
```

Comment:
I suggest changing SnpUtils.getCompoundSnpDistribution(String)  to SnpUtils.getCompoundSnpDistribution(String , 0). 
Even the method of getCompoundSnpDistribution(String) is never used, but getCompoundSnpDistribution(String, int) is used. It is good to test when the second value is 0.